src/Chain.jl

```diff
@@ -58,4 +58,20 @@ Base.getindex(ltc::Chain{L, Periodic}, x::Int) where L = mod1(x, L)
 (ltc::Chain{L, Fixed})(x::Int) where L = 1 <= x <= L ? x : throw(BoundsError(ltc, x))
 (ltc::Chain{L, Periodic})(x::Int) where L = mod1(x, L)
 
+Lattices.neighbors(ltc::Chain{L, Periodic}, s::Int; length=1) where L = (ltc(s+length), ltc(s-length))
+
+# OPT: Can we do something about the type instability here?
```

Comment:
I think `Union{Nothing, Int64}` should be fine (this is optimized after 1.0). the `iterate` function also have `Union{Nothing, blabla}`